includes/class-press-sync.php

```diff
@@ -1376,4 +1383,62 @@ public function maybe_remove_post_id( $object_args ) {
 
 		return $object_args;
 	}
+
+	/**
+	 * Counts the number of taxonomies to sync.
+	 *
+	 * @since NEXT
+	 * @return int
+	 */
+	public function count_taxonomy_term_to_sync() {
+		$sql = <<<SQL
+SELECT DISTINCT
+	taxonomy, term_id
+FROM
+	{$GLOBALS['wpdb']->term_taxonomy}
```

Comment:
Not going to block this, but when we look at refactoring this, might be cleaner to do global $wpdb; $this->wpdb = $wpdb; in the constructor :)

Reply:
I prefer to keep globals in `$GLOBALS` so it's explicitly clear that it's global. I know, as WP developers, that $wpdb is always assumed to be from global scope but I think this is good form for someone who might not be aware.